quotes.json

```
@@ -699,6 +699,12 @@
     "author": "Betty Friedan",
     "text": "It is easier to live through someone else than to become complete yourself."
   },
+  {
+    "author": "Bill Gates",
+    "source": "https://www.goodreads.com/author/quotes/23470.Bill_Gates",
```

Hello and thank you very much for your contribution! 🙏 

I think we should use a direct link to the quote to make it more clear (as stated in the project contribution guide).

```suggestion
    "source": "https://www.goodreads.com/quotes/323288",
```